src/Table/index.tsx

```diff
@@ -27,3 +27,8 @@ export function Table<
 >(props: TableProps<RecordType>) {
   return <StyledTable rowKey="id" {...props} />;
 }
+
+// Re-define JSX API components, see https://github.com/ant-design/ant-design/blob/master/components/table/Table.tsx#L548
```

Comment:
```suggestion
// Re-define JSX API components, see https://github.com/ant-design/ant-design/blob/a4d139687a6ca5a0abad09ac5cdadce617ccf9ca/components/table/Table.tsx#L548-L550
```

Changed to a permalink with a range of lines

Reply:
Makes sense, thanks